test/inv.jl

```diff
@@ -55,8 +55,8 @@ end
         1.63327582123091175e-01 1.70552365412100865e-01 4.55720450934200327e-01 1.23299968650232419e-01
     ]
     sm = SMatrix{4,4}(m)
-    @test norm(Matrix(sm*inv(sm) - eye(4))) < 10*norm(m*inv(m) - eye(4))
-    @test norm(Matrix(inv(sm)*sm - eye(4))) < 10*norm(inv(m)*m - eye(4))
+    @test norm(Matrix(sm*inv(sm) - eye(4))) < 12*norm(m*inv(m) - eye(4))
+    @test norm(Matrix(inv(sm)*sm - eye(4))) < 12*norm(inv(m)*m - eye(4))
```

Comment:
This change is ok I think. There's nothing special about the factor of `10` which you've replaced with 12 here.  I'd say some difference is to be expected with a switch in algorithms.